tardis/montecarlo/src/cmontecarlo.c

```diff
@@ -364,8 +372,35 @@ increment_j_blue_estimator (const rpacket_t * packet, storage_model_t * storage,
 #ifdef WITHOPENMP
 #pragma omp atomic
 #endif
+
   storage->line_lists_j_blues[j_blue_idx] +=
```

Comment:
@SooryaN - try to figure out what this instruction does and why we are doing it. Adding a **line-specific** counter is then a matter of a few lines...